ROADMAP.md

```diff
@@ -1,39 +1,78 @@
 # Roadmap
 
-| Sandboxer      | Sandbox          | 2023 H1 | 2023 H2 | 2024 H1 |
-|----------------|------------------|---------|---------|---------|
-| **MicroVM**    | Cloud Hypervisor | ✓       |         |         |
-|                | QEMU             | ✓       |         |         |
-|                | StratoVirt       | ✓       |         |         |
-|                | Firecracker      |         |         | ✓       |
-| **App Kernel** | Quark            | ✓       |         |         |
-|                | gVisor           |         |         | ✓       |
-| **Wasm**       | WasmEdge         | ✓       |         |         |
-|                | Wasmtime         |         | ✓       |         |
-| **runC**       | runC             |         | ✓       |         |
-
+This document defines a high level roadmap for Kuasar development.
 
 ## 2023 H1
 
-+ Support parts of mainstream sandbox, i.e. Cloud Hypervisor, QEMU, StratoVirt, WasmEdge, Quark
-+ Support connection of containerd and iSulad via Sandbox API
+### Core framework
+
+#### Sandbox API
++ Define and develop new Sandbox API with containerd team
+
+#### Integration of high-level container runtime
++ Containerd (of Kuasar community)
++ Containerd (of Containerd community with kuasar-shim)
++ iSulad
+
+#### Sandbox and container management
++ Cloud Hypervisor
++ QEMU
++ StratoVirt
++ WasmEdge
++ QuarkContainer
+
+### Test
++ Performance testing towards on startup time and memory overhead of kuasar vmm sandbox
 
 ## 2023 H2
 
-+ Support runC container
-+ Support Wasmtime
-+ Support DRA(Dynamic Resource Allocation) API
-+ Support CDI(Container Device Interface) API
-+ Start the process of donating projects to CNCF
+### Core framework
+
+#### Integration of high-level container runtime
++ Containerd (of Containerd community)
+
+#### Sandbox and container management
++ Wasmtime
++ Runc
+
+#### Kubernetes feature support
++ Support Kubernetes and CDI(Container Device Interface) API
```

Comment:
Include Dynamic Resource Allocation (DRA) and Node Resource Interface (NRI) support?

Reply:
Updated